src/Navigation/Drawer/DrawerStyle.ts

```diff
@@ -44,38 +44,47 @@ const closeToLeft = keyframes`
 const fadeIn = keyframes`
   from {
     opacity: 0;
+    visibility: hidden;
+    background-color: transparent;
   }
-
+  
   to {
     opacity: 1;
+    visibility: visible;
+    background-color: rgba(0, 0, 0, 0.5);
   }
-`;
+  `;
 
 const fadeOut = keyframes`
   from {
     opacity: 1;
+    visibility: visible;
```

Comment:
Not sure about this, as the visibility is set here, does it mean for the `hidden` case we want the drawer to exist but just hidden? When it's faded out it shouldn't be in the display?

Reply:
Yep, it is just hidden when it is "closed". That's essentially how I "fixed" this, because otherwise you have to do more complicated state management to ensure that display is only set to `none` _after_ the close animation is done or _before_ the open animation runs.

Reply:
Oh weird, shouldn't be just hidden when closed 🤔 .